rmm/monitor/src/event/mainloop.rs

```
@@ -55,7 +55,10 @@ impl Mainloop {
                 ctx.resize_ret(ret_num);
                 self.queue.lock().push_back(ctx);
             },
-            || {},
+            || {
+                let ctx = Context::new(rmi::NOT_SUPPORTED_YET);
+                self.queue.lock().push_back(ctx);
+            },
```

Isn't it better to call an explicit panic when unsupported commands have been reached? (as we are easy to miss important log messages).

---

If we panic unsupported commands like `RMI_RTT_FOLD`, it is hard to progress one TC fully even it is not related directly. And we returns errors to host when it comes to come unsupported commands, i think it's better for now.